code/modules/mob/living/simple_animal/abnormality/waw/fire_bird.dm

```diff
@@ -43,7 +43,7 @@
 		In your hands you carry a bow nocked with an arrow, it's your last one. <br>\
 		You've been pursuing your prey for days, you..."
 	observation_choices = list("Fire an arrow", "Take off your blindfold", "Do nothing") //awaiting extra answers functionality //"Take off your blindfold" should be a third option
```

Comment:
Whilst there, please delete the `"Take off your blindfold" should be a third option` comment and maybe put it into the third position.